Enigmatry.Entry.Blueprint.Data.Migrations/Seeding/UserStatusSeeding.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Enigmatry.Entry.Blueprint.Domain.Users;
+
+namespace Enigmatry.Entry.Blueprint.Data.Migrations.Seeding;
+
+internal class UserStatusSeeding() : EntityWithEnumIdSeeding<UserStatus, UserStatusId>(statusId =>
+    new { Id = statusId, statusId.Name, Description = GetDescription(statusId) })
+{
+    private static string GetDescription(UserStatusId statusId) =>
```

Comment:
Description field was added to showcase how to populate extra column for each given enum value.